### DISCUSSION TITLE
Two Renovate configs in a repository

### DISCUSSION THREAD

**Discussion by remal** [renovatebot/renovate#28742] [Request Help] [answered]
### What would you like help with?

Other

### How are you running Renovate?

Mend Renovate hosted app on github.com

### Please tell us more about your question or problem

[This repository](https://github.com/remal-gradle-plugins/template) has two Renovate configs:
* [`renovate.json5`](https://github.com/remal-gradle-plugins/template/blob/main/renovate.json5)
* [`.github/renovate.json5`](https://github.com/remal-gradle-plugins/template/blob/main/.github/renovate.json5)

I need both of the configs. `.github/renovate.json5` is automatically synchronized to repositories created from this template repository, and `renovate.json5` should provide additional Renovate configuration for this repository only.

If I understand correctly, `renovate.json5` has precedence over `.github/renovate.json5`. However, the issue is that `renovate.json5` file was added after `.github/renovate.json5`. And the existence of `.github/renovate.json5` seems to be cached to Renovate.

Am I right that after pruning cache for this repository, Renovate will start looking at `renovate.json5` file and not `.github/renovate.json5`? If yes, could you please prune the cache?

**Comment by rarkins** [comment]
@nabeelsaabna can you delete the cache for https://github.com/remal-gradle-plugins/template ?

**Reply by nabeelsaabna** [reply, marked as answer]
done
This time I renamed the file to keep a backup to check what is the issue

**Reply by rarkins** [reply]
I think it's that:
- Renovate remembers what was the previous config file name it found
- It saves time by checking that it still exists
- This is a rare case where both the "old" and "new" config filenames exist

**Reply by remal** [reply]
It works now. Thanks!

@rarkins, when I looked at the Renovate sources, I came up with the same assumption.

Closing the discussion.
